.github/workflows/unittests.yml

```
@@ -65,7 +65,7 @@ jobs:
           -d '{"event_type": "docker-build",
               "client_payload":{"repository":"marblerun",
                                 "sign":"nightly",
-                                "imagename":"coordinator",
+                                "imagename":"marblerun/coordinator",
```

```suggestion
                                "imagename":"marblerun/coordinator-debug",
```
I think it will build a debug coordinator by default? Also seems reasonable to me to publish nightlies (only) for the debug image.

---

Sounds good